internal/faustjs.org/docs/gutenberg/filters.mdx

```diff
@@ -0,0 +1,131 @@
+---
+slug: /gutenberg/filters
+title: Filters
+description: wp-graphql-content-blocks plugin filters available to you
+---
+
+Sometimes you want to change or add new types of block attributes that you want to be exposed in the WPGraphQL API.
+Below are the different types of filters available to you in the `wp-graphql-content-blocks` plugin.
+
+## wpgraphql_content_blocks_block_class
```

Comment:
I notice that this is the only filter provided with a header. Do we have others that should be given a header here as well that are unique to the `wp-graphql-content-blocks` plugin? If not, I would suggest removing the header here and changing line 8 to something like:

>  This can be accomplished by using the `wpgraphql_content_blocks_block_class` filter available to you in the `wp-graphql-content-blocks` plugin.